projects/storefrontapp/src/app/spartacus/spartacus-b2c-configuration.module.ts

```diff
@@ -6,6 +6,18 @@ import {
   mediaConfig,
   PWAModuleConfig,
 } from '@spartacus/storefront';
+import { environment } from '../../environments/environment';
+
+const defaultBaseSite = [
+  'electronics-spa',
+  'electronics',
+  'apparel-de',
+  'apparel-uk',
+  'apparel-uk-spa',
+];
+const baseSite = environment.epdVisualization
+  ? ['electronics-epdvisualization-spa'].concat(defaultBaseSite)
```

Comment:
When doing QA of the feature, I discovered this basesite is not present in the sample data. Should we remove this line of code?